FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Core/PipelineRegistrationExtensions.cs

```diff
@@ -20,17 +20,15 @@ public static class PipelineRegistrationExtensions
         public static IServiceCollection AddJobScheduler(
             this IServiceCollection services)
         {
-            services.AddSingleton<IJobStore, AzureBlobJobStore>();
+            services.AddSingleton<JobHosting, JobHosting>();
 
-            services.AddSingleton<JobProgressUpdaterFactory, JobProgressUpdaterFactory>();
+            services.AddSingleton<IJobFactory, AzureStorageJobFactory>();
 
-            services.AddSingleton<JobManager, JobManager>();
-
-            services.AddSingleton<IJobExecutor, JobExecutor>();
+            services.AddSingleton<IAzureTableClientFactory, AzureTableClientFactory>();
 
-            services.AddSingleton<JobExecutor, JobExecutor>();
+            services.AddSingleton<JobManager, JobManager>();
 
-            services.AddSingleton<ITaskExecutor, TaskExecutor>();
+            services.AddSingleton<SchedulerService, SchedulerService>();
```

Comment:
Can be simplified to `services.AddSingleton<SchedulerService>()`.

Reply:
Sure, refined.